app/assets/stylesheets/global.scss

```diff
@@ -57,6 +57,7 @@ button, .button {
 .width-250 { width: 250px; }
 .width-300 { width: 300px; }
 .spacer { height: 2px; padding: 0; }
+.spacer-alt { height: 2px; padding: 0; background-color: #8B8687; }
```

Comment:
I would modify these two lines to be like so:

```css
.spacer, .spacer-alt { height: 2px; padding: 0; }
.spacer-alt { background-color: #8B8687; }
```

That is, keep the layout structure the same between them, forcibly (so nobody accidentally changes one and misses the other), and then style the `spacer-alt` with a different color after.